src/lando/main/models/repo.py

```diff
@@ -118,7 +118,7 @@ def scm(self) -> AbstractSCM:
         return self._scm
 
     def __str__(self):
-        return f"{self.name} ({self.default_branch})"
+        return f"{self.name} ({self.scm_type})"
```

Comment:
We may want to keep the default branch visible if present here, as this will help differentiating Git repos as long as they are in Repo+Branch mode.